app/controllers/web/console.php

```diff
@@ -14,6 +14,7 @@
     $layout
         ->setParam('description', 'Appwrite Console allows you to easily manage, monitor, and control your entire backend API and tools.')
         ->setParam('analytics', 'UA-26264668-5')
+        ->setParam('analytics-option', App::getEnv('_APP_OPTIONS_ANALYTICS', 'enabled'))
```

Comment:
`analytics-enabled` with value `true` or `false` makes more sense.

Reply:
Thanks for the recommendation! I will change that in the course of further suggestions.